repose-aggregator/src/docs/asciidoc/filters/api-validator.adoc

```diff
@@ -153,7 +153,7 @@ This configuration expands the basic example in order to show off all of the fea
     Triggers off of `X-Roles` header. +
     *DEPRECATED:* Roles defined outside of the WADL will not be supported in *Repose* 9 and this attribute will not be available.
 <2> If the api-validator config `multi-match` is set to `true` then the default validator will be the first validator to process the incoming request.
-    If multi-match is set to `false` and if no validator is matched to the users' roles, then the filter will use the default validator. (Default: false) +
+    If multi-match is set to `false` and if no validator is matched to the users' roles, then the filter will use the default validator. (Default: `false`) +
```

Comment:
can you pull all these unrelated doc changes into a separate pr?

Reply:
These were all pulled out to #1838.